Unify ffmpeg commands for stream and download

oxzi opened the pull request:
Previously, two ffmpeg commands were executed if the download flag was
set. These two commands had hardcoded the same parameters. Further,
since at least for the input they had the same task, they unnecessarily
caused a lot of load. By unifying into one command with two outputs, the
load on my test system dropped noticeably.

mtsonline commented on stream.py:
```diff
@@ -236,21 +235,34 @@ def stream_intro():
-    logging.info("streaming meeting...")
-    p = subprocess.call(ffmpeg_args)
+    ffmpeg_outputs = []
+    if args.download:
```
Thanks for your improovement. could you add a note, that the meeting is saved as file / streamed as it was in the original - so one can see in the logging, if download / streaming was active?

oxzi replied:
Thanks for your review.
I have just added the same logging messages which were used before. Please take another look.

oxzi committed:
Unify ffmpeg commands for stream and download

Previously, two ffmpeg commands were executed if the download flag was
set. These two commands had hardcoded the same parameters. Further,
since at least for the input they had the same task, they unnecessarily
caused a lot of load. By unifying into one command with two outputs, the
load on my test system dropped noticeably.

mtsonline commented:
Thanks again for this contribution!
